### PR TITLE
Fix: computed setter was missing

### PR DESCRIPTION
**Pull request by elisegriset92**

### CODE REVIEW

File: _dev/src/components/product-feed/settings/summary/summary.vue [hunk 1]
```diff
@@ -249,8 +249,13 @@ export default {
...
   computed: {
-    disabledExportButton() {
-      return !(this.acceptSyncSchedule && this.understandTerms);
+    disabledExportButton: {
+      get() {
+        return !(this.acceptSyncSchedule && this.understandTerms);
+      },
+      set(value) {
+        return value;
+      },
```
Comment (Quetzacoalt91) [inline _dev/src/components/product-feed/settings/summary/summary.vue:258]: There is a better way to fix the issue here. If you want to manage a variable inside your component, it should be defined in the `data()`, not as a computed.

Comment (elisegriset92) [reply _dev/src/components/product-feed/settings/summary/summary.vue:258]: If I put it in the data() then it's not dynamically updated when the other data variables are modified

Comment (Quetzacoalt91) [reply _dev/src/components/product-feed/settings/summary/summary.vue:258]: Then this computed should depend on another variable you will create, something like `loading` or `apiCallInProgress` from my understanding.

Comment (elisegriset92) [reply _dev/src/components/product-feed/settings/summary/summary.vue:258]: It is depending on the state of 2 checkboxes. I don't understand your point with loading/apiCall

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**